src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+  - OpenTelemetry
+metaDescription: You can set up OpenTelemetry to monitor a simple Java app.
+---
+
+import opentelemetrySpringBootASCIIDisplay from 'images/opentelemetry_screenshot-crop_Spring-Boot-ASCII-display.png'
+
+import opentelemetryMainDistributedTracingPage from 'images/opentelemetry_screenshot-crop_main-distributed-tracing-page.png'
+
+import opentelemetryDistributedTracingError from 'images/opentelemetry_screenshot-crop_distributed-tracing-error.png'
+
+import opentelemetryInProcessSpans from 'images/opentelemetry_screenshot-crop_in-process-spans.png'
+
+import opentelemetryExceptiononSpanEvent from 'images/opentelemetry_screenshot-crop_exception-on-span-event.png'
+
+import opentelemetryAttributesTab from 'images/opentelemetry_screenshot-crop_attributes-tab.png'
+
+import opentelemetryMetricsExplorerMain from 'images/opentelemetry_screenshot-crop_metrics-explorer-main.png'
+
+import opentelemetryMetricsExplorerDimensions from 'images/opentelemetry_screenshot-crop_metrics-explorer-dimensions.png'
+
+import opentelemetryJvmsPageMain from 'images/opentelemetry_screenshot-crop_jvms-page-main.png'
+
+import opentelemetryCompareJvmInstances from 'images/opentelemetry_screenshot-full_compare-jvm-instances.png'
+
+import opentelemetryJvmColorCodedComparison from 'images/opentelemetry_screenshot-full_jvm-color-coded-comparison.png'
+
+import opentelemetryMainLogsPage from 'images/opentelemetry_screenshot-full_main-logs-page.png'
+
+import opentelemetryLogsinTerminal from 'images/opentelemetry_screenshot-full_logs-in-terminal.png'
+
+import opentelemetryLogDetailsWithTrace from 'images/opentelemetry_screenshot-full_log-details-with-trace.png'
+
+import opentelemetryTraceRelatedtoLog from 'images/opentelemetry_screenshot-full_trace-related-to-log.png'
+
+import opentelemetryTraceWaterfallFromLog from 'images/opentelemetry_screenshot-full_trace-waterfall-from-log.png'
+
+import opentelemetryLogCapturedinTraceSpan from 'images/opentelemetry_screenshot-full_log-captured-in-trace-span.png'
+
+
+
+
+
+Try out this Java tutorial to see what the New Relic platform can do with your OTLP data. You can quickly learn how to configure your OpenTelemetry SDK to optimize your observability experience in New Relic. You'll do all this using a demo Spring application that generates the nth number in the Fibonacci sequence.
+
+You'll find three tracks below to help you get acquainted with New Relic and OpenTelmentry. Try out the tracks that are interesting to you:
+
+* [Run our pre-instrumented program:](#pre-instrum) This is the fastest way to send some demo data to New Relic and see how it is displayed in the UI. This demo app has pre-loaded instrumentation and SDK configurations that follow our [best practices](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-overview/) to generate and export metrics, logs, and traces. You can inspect our code and apply relevant sections to your own apps.
```

Comment:
I might make it more clear that all these work with the same demo app in different ways. The 'program' in the first one is throwing me off a bit, as it's not clear how that applies to 'app'. Also, the second entry doesn't mention an 'app' or 'program'. In short, wondering if it could be more aligned language-wise to easily communicate: they all use this demo app, but here's the different ways they use them.

Reply:
@zuluecho9 In addition to changing the intro to clarify that all the tutorials use the same demo app, I changed the run-in headings:

* Run the pre-instrumented demo app
* Monitor the demo app with the OpenTelemetry Java agent
* Set up the demo app manually